govc/host/vnic/hint.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright (c) 2022 VMware, Inc. All Rights Reserved.
```

Comment:
The header check wants `2022` to be `2022-2022`
And there's also some whitespace formatting lost. Suggest you just copy-n-paste the header from another file and adjust the year.